src/components/editMeasure/testCases/components/editor/Editor.tsx

```diff
@@ -65,6 +64,36 @@ const Editor = ({
     },
   });
 
+  const toggleSearchBox = useCallback(() => {
```

Comment:
hard to cover this callback as test case is not working for `execCommand`. could be because it is deprecated.

Reply:
May have something to do with the way the imports for searchbox are brought in. But it doesn't seem worth the time to find

Reply:
yeah